src/plugin.ts

```diff
@@ -0,0 +1,28 @@
+import { UserOptions, defaultOptions, Options } from './options'
+import { stringifiedRequestAttributes } from './formatting'
+import { ApolloServerPlugin } from 'apollo-server-plugin-base'
+
+export default (ops: UserOptions = {}): ApolloServerPlugin => {
+  const options: Options = Object.assign(defaultOptions, ops);
+  const log = (msg: string): void => {
+    options.logger(`${options.prefix()}${msg}`)
+  }
+  return {
+    requestDidStart(r) {
+      const isInspectionQuery = r.operationName === 'IntrospectionQuery'
```

Comment:
r.operationName is a different property than in extension api. you probably need to use r.request.operationName